README.md

```diff
@@ -129,6 +142,30 @@ There are two parameters:
 * a space number (it could just as easily have been a space name), and
 * a tuple.
 
+### msgpack.v5 migration
+
+Most function names and argument types in `msgpack.v5` and `msgpack.v2`
+have not changed (except at least `EncodeInt`, `EncodeUint` and `RegisterExt`).
```

Comment:
I don't get what `except at least` should mean. It is a confusing sentence.

Reply:
I've rewrote the sentence:
```
Most function names and argument types in `msgpack.v5` and `msgpack.v2`         
have not changed (in our code, we noticed changes in `EncodeInt`, `EncodeUint`  
and `RegisterExt`). But there are a lot of changes in a logic of encoding and   
deconding. On the plus side the migration seems easy, but on the minus side you 
need to be very careful.
```

I hope that it became clearer.